pypom/page.py

```diff
@@ -107,13 +107,31 @@ def open(self):
         raise UsageError('Set a base URL or URL_TEMPLATE to open this page.')
 
     def wait_for_page_to_load(self):
-        """Wait for the page to load.
+        """
+        Wait for the page to load.
```

Comment:
Please put the summary back on the first line. See [PEP 257](https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings).